usefulUtilities/airAndWhiteboardDraw/entityResources/entity/whiteboard.json

```diff
@@ -0,0 +1,638 @@
+{
+  "DataVersion": 0,
+  "Entities": [
+    {
+    "id": "{4d202310-282f-43b7-bb4c-e57d2cd2e46e}",
+    "type": "Text",
+    "lastEdited": 1555007902289146,
+    "parentID": "{f8b95e5a-a886-4606-b023-82e0e026d33f}",
+    "name": "Whiteboard Reset",
+    "locked": true,
+    "position": {
+        "x": 0.5075650215148926,
+        "y": -0.591064453125,
+        "z": -0.00726318359375
+    },
+    "dimensions": {
+        "x": 0.13745681941509247,
+        "y": 0.06351786851882935,
+        "z": 0.009999999776482582
+    },
+    "rotation": {
+        "x": -0.0000152587890625,
+        "y": 1,
+        "z": -0.0000457763671875,
+        "w": 0.0000762939453125
+    },
+    "created": 1554422118124390,
+    "lastEditedBy": "{6cb93832-459d-4e8c-a721-c02d692279f9}",
+    "queryAACube": {
+        "x": -71.7729721069336,
+        "y": 890.8225708007812,
+        "z": -7.1639909744262695,
+        "scale": 0.1517527550458908
+    },
+    "grab": {
+        "grabbable": false,
+        "equippableLeftRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        },
+        "equippableRightRotation": {
+            "x": -0.0000152587890625,
+            "y": -0.0000152587890625,
+            "z": -0.0000152587890625,
+            "w": 1
+        }
+    },
+    "damping": 0,
+    "angularDamping": 0,
+    "collisionMask": 7,
+    "collidesWith": "static,dynamic,kinematic,",
+    "script": "http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/airAndWhiteboardDraw/entityResources/entityClientScripts/whiteboardReset.js",
+    "scriptTimestamp": 1554489729490,
+    "text": "Reset",
+    "lineHeight": 0.05999999865889549,
+    "clientOnly": false,
+    "avatarEntity": false,
+    "localEntity": false,
+    "faceCamera": false,
+    "isFacingAvatar": false
+},
+    {
+    "id": "{7535a63a-1d80-47b5-844a-4d11514326a9}",
+    "type": "Box",
+    "lastEdited": 1554923712547407,
+    "parentID": "{f8b95e5a-a886-4606-b023-82e0e026d33f}",
+    "name": "Whiteboard Palette Square",
+    "locked": true,
+    "userData": "{\"textureURL\":\"http://hifi-content.s3-us-west-1.amazonaws.com/Experiences/Releases/usefulUtilities/airAndWhiteboardDraw/entityResources/resources/textures/cyan.png\"}",
```

Comment:
please replace this link with a relative link (which you'll then have to update upon release)